tests/utils.py

```diff
@@ -24,11 +24,15 @@
 def add_version_constraint_to_project(project_dir: py.path.local):
     constraints = os.environ.get("INMANTA_LSM_MODULE_CONSTRAINTS", "")
     if constraints:
-        with open(project_dir / "module.yml", "r") as fh:
-            module_config = yaml.safe_load(fh)
-            module_config["requires"] = constraints.split(";")
-        with open(project_dir / "module.yml", "w") as fh:
-            yaml.dump(module_config, fh)
+        if os.path.exists(project_dir / "module.yml"):
+            with open(project_dir / "module.yml", "r") as fh:
+                module_config = yaml.safe_load(fh)
+                module_config["requires"] = constraints.split(";")
+            with open(project_dir / "module.yml", "w") as fh:
+                yaml.dump(module_config, fh)
+        else:
+            with open(project_dir / "requirements.txt", "a") as fh:
```

Comment:
Oh, this won't work, my bad. V2 modules ignore `requirements.txt`, we need to add it to `setup.cfg`.